modules/compliance-scansetting-cr.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+:_content-type: CONCEPT
+[id="compliance-scansetting-cr"]
+= `ScanSetting` Custom Resource
+
+The `ScanSetting` Custom Resource now allows you to override the default CPU and memory limits of scanner pods through the scan limits attribute. The compliance operator will use defaults of 500Mi memory, 100m CPU for the scanner container, and 200Mi memory with 100m CPU for the `api-resource-collector` container. To set the memory limits of the operator, modify the `Subscription` object if installed through OLM or the operator deployment itself.
+
+To increase the default CPU and memory limits of the Compliance Operator, see _Increasing Compliance Operator resource limits_.
+
+[IMPORTANT]
+====
+Increasing the memory limit for the Compliance Operator or the scanner pods is needed if the default limits are not sufficient and the Operator or scanner pods are killed by the Out Of Memory (OOM) killer process.
```

Comment:
> 🚫  kill verb
Do not use except when you are documenting a UNIX system, such as AIX. Use "end" or "stop".